files/nrpe/check_object.py

```diff
@@ -277,12 +376,12 @@ def parse_command_line():
   parser.add_option("-p", "--password", dest='password', help='password')
   parser.add_option("-s", "--s3_access_key", dest='s3_access_key', help='username')
   parser.add_option("-k", "--s3_secret_key", dest='s3_secret_key', help='password')
-  parser.add_option("-l", "--s3_host", dest='s3_host', help='host')
+  parser.add_option("-b", "--s3_bucket_url", dest='s3_bucket_url', help='URL to S3 bucket')
+  parser.add_option("-l", "--s3_host", dest='s3_host', help='s3 host')
   parser.add_option("-t", "--tenant", dest='tenant', help='tenant name')
-
   parser.add_option("-d", "--debug", dest='debug', action='store_true', help='Debug mode. Enables logging')
-
-  parser.add_option("-b", "--s3_bucket_url", dest='s3_bucket_url', help='URL to S3 bucket')
+  parser.add_option("-w", "--swift_host", dest='swift_host', help='swift host')
+  parser.add_option("-c", "--swift_bucket_url", dest='swift_bucket_url', help='URL to swift bucket')
```

Comment:
The -b took the full URL https://blah/blah/cat.jpg
the URL to swift bucket is unintuitive as I believe you are wanting bucket/cat.jpg 
so maybe update the help label to reflect that

Reply:
I have updated the documentation and the name of the variable.